apps/studio/components/interfaces/Linter/LinterDataGrid.tsx

```diff
@@ -176,7 +181,7 @@ const LinterDataGrid = ({
               </div>
               <Button type="text" icon={<X />} onClick={handleSidepanelClose} />
             </div>
-            <div className="p-6">
+            <div className="p-6 flex-grow min-h-0 overflow-y-auto">
```

Comment:
This fixes the issue where content wasn’t scrollable. It’s analogous to QueryPerformanceGrid